client/modules/IDE/components/NewFileForm.jsx

```diff
@@ -30,14 +31,14 @@ class NewFileForm extends React.Component {
             className="new-file-form__name-input"
             id="name"
             type="text"
-            placeholder="Name"
+            placeholder={i18n.t('NewFileForm.Placeholder')}
```

Comment:
Same comments as in #1537. In components, let's try and use _withTranslation_. 

The `validate()` function is a good case for using the `i18next.t()` function directly but here it should be `this.props.t()`.

Reply:
@andrewn I made the changes requested. The problematic was also present in the Folder modal